tools/ci/testlist/sim-02.dat

```diff
@@ -1,5 +1,8 @@
 /sim/*/*/*/[o-z]*
 
+# heavy test, moved to a separate set
+-sim:ostest
```

Comment:
why need to move the new test list?

Reply:
just because it's heavier than other tests.

Reply:
But, I don't think there is any real benefit here. And how to extend with new test? e.g. if we add qemu for arm/xtensa later, how we organize the test list?

Reply:
my assumption: these lists were organized to make each jobs to take similar time.
if we add tests, they should be organized according to how long they take.

maybe my assumption is wrong? please correct me in that case.

Reply:
The name of sim-ostest.dat is too specific, which mean we can only put one test sim:ostest into this list, but the time to execute ostest is around 1 minute and much smaller than the average value.
I don't know how many test will be added later, but the time to execute the test is short in the current stage. Why not postpone the change?

Reply:
ok. i will drop the patch for now.

Reply:
But, I don't see the related change.

Reply:
i haven't made the change yet.

Reply:
done.